source/banner.html.erb

```diff
@@ -0,0 +1,18 @@
+<section class="banner">
+  <div class="container">
+    <h1 class="banner-logo">
```

Comment:
I anticipate this only having one page, but that's a good way to look at it. Also, ordered, will read. Thanks!